### DISCUSSION TITLE
Task worker input

### DISCUSSION THREAD

**Discussion by Mishtir** [Netflix/conductor#3081] [Q&A]
Hi,
can we pass task worker input through an api in conductor? 
So I have a requirement to pass task input parameter from UI field , not hard coded in json file. How can we pass the input to task work through an api call.?

**Comment by dougsillars** [comment]
Hi @Mishtir,

Absolutely.  If you have a WAIT/HUMAN task waiting for customer input, you can complete the wait task via the API with output parameters:

```shell
curl -X POST "https://play.orkes.io/api/tasks" \
-H  "accept: text/plain" \
-H  "Content-Type: application/json" \
-d '{"workflowInstanceId":"<workflowId","taskId":"taskID","outputData": {"cake":"yummy"},"reasonForIncompletion":"NA","status":"COMPLETED"}'
```

This will add {"cake":"yummy"} to the output of the task.
